front/ticket.php

```diff
@@ -46,10 +46,12 @@
 
 if ($_SESSION['glpirefresh_ticket_list'] > 0) {
    // Refresh automatique  sur tracking.php
-   echo "<script type=\"text/javascript\">\n";
-   echo "setInterval(\"window.location.reload()\",".
-         (60000 * $_SESSION['glpirefresh_ticket_list']).");\n";
-   echo "</script>\n";
+    echo "<script type=\"text/javascript\">\n";
+    echo "setTimeout(function(){ \n".
+        "window.location.href = window.location.href; }, \n".
```

Comment:
I made an update from version 0.85 to 9.1.2 and found this problem. The page of tickets was sending requests without respecting the 3 minutes that had configured in the screen of customization. It did not show any ticket and also did not allow to click anything because it sent requests to each section for tickets.php
Window.location.reload () has been retained.